modules/rdma-disabling-irdma-kernel-module.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * hardware_accelerators/rdma-remote-direct-memory-access.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rdma-disabling-irdma-kernel-module_{context}"]
+
+= Disabling the IRDMA kernel module
+
+On some systems, including the DellR750xa, the IRDMA kernel module creates problems for the NVIDIA Network Operator when unloading and loading the DOCA drivers. Use the following procedure to disable the module.
+
+.Procedure
+
+. Generate the following machine configuration file:
++
+[source,terminal]
+----
+$ cat <<EOF > 99-machine-config-blacklist-irdma.yaml 
```

Comment:
Similar question about whether this command is appropriate to exist on its own without a bunch of YAML below it to add to the created file, but again maybe I do not understand